client/app/components/TestReactRouterRedirect.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default class TestReactRouterRedirect extends React.Component {
+
+  static checkAuth(nextState, replaceState) {
+    const notAuthorized = true;
+    if (notAuthorized) {
+      replaceState({ redirectFrom: nextState.location.pathname }, '/');
```

Comment:
This puts the object in the "state" of the routerRedirect object that the callback will get

    //       routeRedirect: {
    //         pathname: "Path <String>",
    //         search: "Query <String>",
    //         hash: "Hash <string>",
    //         state: "Custom State <Object>",
    //         ...
    //        }